js/packages/web/src/pages/index.tsx

```diff
@@ -45,12 +45,12 @@ export async function getServerSideProps(context: NextPageContext) {
     return { props: { storefront } };
   }
 
-  return {
+  return { // should we wrap this in props? ({props: {notFound: true}})
     notFound: true,
   };
 }
```

Comment:
notfound and props are on the same level. This is correct but you can check the types and docs.